systems/plants/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ if (eigen3_FOUND)
   add_rbm_mex(forwardKinmex)
   add_rbm_mex(bodyKinmex)
   add_rbm_mex(collisionDetectmex)
+  add_rbm_mex(surfaceTangentsmex)
```

Comment:
this should be a call to add_mex, not add_rbm_mex, because it does not need to link against the rbm objects.